### PR TITLE
fix: proper parsing of input parameters

### PR DESCRIPTION
**Pull request by mgechev**
The input parser was not processing properly negative numbers, which was
causing the bot to exit early.

### CODE REVIEW

File: github-actions/feature-request/src/get-input.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    return (result === 'true' ? true : false) as T;
+  }
+  if (!/^-?\d+(\.?\d*)$/.test(result)) {
+    return result as T;
+  }
+  const num = parseFloat(result);
+  if (isNaN(num)) {
+    throw new Error(`Can't parse ${name} as a numeric value.`);
+  }
+  return num as T;
```
Comment (josephperrott) [inline github-actions/feature-request/src/get-input.ts:27]:
```suggestion
  if (!isNaN(result)) {
    return parseFloat(result) as T;
  }
  return result as T;
```

**Review by josephperrott — APPROVED** [review]
Just one suggestion as an alternative way to parse.

File: github-actions/feature-request/src/get-input.ts (hunk 1 above)
Comment (mgechev) [reply github-actions/feature-request/src/get-input.ts:27]: Interesting! Did not know `isNaN` casts to number if we pass a string.

Comment (mgechev) [reply github-actions/feature-request/src/get-input.ts:27]: TypeScript is unhappy though because `isNaN` expects a number.

**Pull request merged**